siriuspy/siriuspy/devices/bpm_eq.py

```diff
@@ -0,0 +1,541 @@
+"""."""
+import time as _time
+import logging as _logging
+
+import numpy as _np
+import matplotlib.pyplot as _mplt
+
+from mathphys.functions import save_pickle as _savep, load_pickle as _loadp, \
+    get_namedtuple as _namedtuple
```

Comment:
save and load not being used

Reply:
thanks! I used the new `load` and `save` methods from mathphys and created methods `save_data` and `load_data` with them.